### DISCUSSION TITLE
Conflict problem between actions and commands

### DISCUSSION THREAD

**Discussion by moayedalhagy** [def-studio/telegraph#358] [Ideas]
**i'm callback action however you can call me too by a typing /dismiss command**

![image](https://user-images.githubusercontent.com/118882102/224705246-61d1428f-b9c0-45c7-84e1-e5af09cc8170.png)
![image](https://user-images.githubusercontent.com/118882102/224706230-42231983-75e0-400d-8515-216fcd21f78d.png)

I think it would be better to write a new way to invoke Actions in a different way than Commands.

With all due respect
Thanks

**Comment by fabio-ivona** [comment]
We'll work on it, thanks!
